partiql-cli/build.gradle.kts

```diff
@@ -62,6 +61,26 @@ tasks.register<GradleBuild>("install") {
     tasks = listOf("assembleDist", "distZip", "installDist")
 }
 
+tasks.named("build").configure {
+    dependsOn("generateMockDBJAR")
+}
+
+val mockDbJarPath = "$buildDir/tmp/plugins/mockdb"
+val mockDbJarFile = "$buildDir/tmp/plugins"
+
+tasks.register<Jar>("generateMockDBJAR") {
+    dependsOn(":plugins:partiql-mockdb:build")
+    archiveBaseName.set("mockdb")
+    archiveVersion.set("0.1.0")
```

Comment:
Have you considered using [Gradle copy](https://docs.gradle.org/current/userguide/working_with_files.html#sec:copying_single_file_example) instead?